components/support/ktx/src/main/java/mozilla/components/support/ktx/android/org/json/JSONObject.kt

```diff
@@ -11,34 +11,19 @@ import java.util.TreeMap
  * Returns the value mapped by {@code key} if it exists, and
  * if the value returned is not null. If it's null, it returns null
  */
-fun JSONObject.tryGetString(key: String): String? {
-    if (!isNull(key)) {
-        return getString(key)
-    }
-    return null
-}
+fun JSONObject.tryGetString(key: String): String? = optString(key, null)
 
 /**
  * Returns the value mapped by {@code key} if it exists, and
  * if the value returned is not null. If it's null, it returns null
  */
-fun JSONObject.tryGetInt(key: String): Int? {
-    if (!isNull(key)) {
-        return getInt(key)
-    }
-    return null
-}
+fun JSONObject.tryGetInt(key: String): Int? = if (isNull(key)) null else getInt(key)
```

Comment:
Nit: This will still throw a `JSONException` if the value is non-null and cannot be coerced into an `Int`.

If `org.json.JSON` is visible, you may be better off doing: 

```kotlin
fun JSONObject.tryGetInt(key: String): Int? = JSON.toInteger(opt(key))
```

Reply:
Looks like `org.json.JSON` is not visible. I'm going to leave the current implementation since its what we had before.